src/AWSS3.jl

```diff
@@ -85,7 +86,7 @@ function s3_get(
             "return_stream" => return_stream,
         )
 
-        if !isempty(version)
+        if !isnothing(version)
```

Comment:
Making this change non-breaking
```suggestion
        if !isnothing(version) || !isempty(version)
```